jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/FrameFlusher.java

```diff
@@ -235,6 +263,10 @@ public void onCompleteFailure(Throwable failure)
             entry.release();
         }
         entries.clear();
+        if (endPoint instanceof AbstractEndPoint)
+            ((AbstractEndPoint)endPoint).close(failure);
+        else
+            endPoint.close();
```

Comment:
Now that you have `EndPoint.close(Throwable)`, this `instanceof` is not needed anymore.

Reply:
@lachlan-roberts Do you want me to make this change in this branch to save making a new branch and PR?